frontend/src/common/ModelImageList.tsx

```diff
@@ -32,9 +32,11 @@ export default function ModelImageList({ model, value, setImages }: ModelImageLi
     setFlattenedImageList(updatedImageList)
   }, [modelImages])
 
-  const sortedImageListByName = useMemo(() => {
-    return flattenedImageList.sort((a, b) => a.name.localeCompare(b.name))
-  }, [flattenedImageList])
+  const sortByNameAscending = <T extends { name: string }>(a: T, b: T) => {
```

Comment:
If this is left inside this component it should also be in the `useMemo` dependency array. That would cause it to infinitely rerender (as functions are given a new reference on every render), so we should instead move this function outside of the component.